source/mainnet/net/resources/glossary.rst

```diff
@@ -362,6 +369,13 @@ May refer to:
    :ref:`epoch<glossary-epoch>` that is used to seed the :ref:`leader election<glossary-leader-election>` process.
 -  :ref:`Transaction sequence number<glossary-transaction-sequence-number>` (same as account sequence number)
 
+.. _glossary-non-membership-proof:
+
+Non-membership proof
+====================
+
+A proof to determine that a user does not meet one or more attributes, for example, that they are a resident of a country under trade sanctions.
```

Comment:
```suggestion
A proof to determine if an attribute of a users identity is **not** included in a set, for example, that they are **not** a resident of a country under trade sanctions.
```

Reply:
I don't agree with this rephrasing. A proof of the non-membership statement determines that the user's attribute is not a member. Not whether they are a member. If the proof fails to verify then it does not mean that the attribute is not a member of the set. There could be other problems with the proof.

Reply:
I don't get it. Either you are a member, or you're not a member? What falls inbetween or outside this in your opinion, and how would you phrase this?

Reply:
My objection was to the phrasing "if". The only time a proof determines something is when it is valid. If it is not valid it does not mean that the statement is not true.

```
A proof to determine that an attribute of a user's identity is **not** included in a set, for example, that they are **not** a resident of a country under trade sanctions.
```

Reply:
Ah, right.